Detectors/TPC/qc/include/TPCQC/CalPadWrapper.h

```diff
@@ -0,0 +1,60 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef O2_CalPadWrapper_H
+#define O2_CalPadWrapper_H
+
+#include <TClass.h>
+#include <TObject.h>
+
+#include "TPCBase/CalDet.h"
+
+namespace o2
+{
+namespace tpc
+{
+namespace qc
+{
+/// A wrapper class to easily promote any type to a TObject
+/// does not take ownership of wrapped object and should not be used
+/// in tight loops since construction expensive
```

Comment:
Please add a statement that this is a temporary solution until QualityControl can handle objects which don't inherit from TObject.